src/main/java/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,34 @@
+package dao;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+import model.OperationType;
+
+public class FileReaderImpl implements dao.FileReader {
+
+    @Override
+    public List<FruitTransaction> readFile(String fileName) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            br.readLine();
```

Comment:
You are skipping the first line assuming it to be a header. It would be better to check if the line is a header (for example, by checking if it contains column names) before skipping it.